map/traffic_manager.hpp

```diff
@@ -49,8 +57,24 @@ class TrafficManager
   void CalculateSegmentsColoring(traffic::TrafficInfo const & trafficInfo,
                                  df::TrafficSegmentsColoring & output) const;
 
+  static void ThreadRoutine(TrafficManager * manager);
+  void WaitForRequest(list<MwmSet::MwmId> & mwms);
+  void RequestTrafficData(MwmSet::MwmId const & mwmId);
```

Comment:
This method is called "Request...", but it doesn't return anything. Moreover, it looks like that it actually updates cached geometry and calls UpdateTraffic for a country. Why not to rename this method? How about DrawTrafficForMwm?